### PR TITLE
Support https protocol (testcafe #1985)

### PR DESCRIPTION
**Pull request by intermike**

### CODE REVIEW

File: test/client/fixtures/utils/url-test.js [hunk 1]
```diff
@@ -211,6 +216,12 @@ test('relative path', function () {
     strictEqual(parsedUrl.partAfterHost, 'share?id=1kjQMWh7IcHdTBbTv6otRvCGYr-p02q206M7aR7dmog0');
 });
 
+test('relative path (https proxy)', function () {
```
Comment (miherlosev) [inline test/client/fixtures/utils/url-test.js]: Change the structure of tests:
```js
module('https proxy protocol');
test('relative path');
test('special pages');
test('http');
```

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Pull request merged**